website/docs/props.md

```diff
@@ -0,0 +1,47 @@
+Add any property to your visual studio project
+This allows you to set properties that premake does not support without extending it
+
+Values set at one time are sorted alphabetically
```

Comment:
if order is important for msvc, adding in the doc the way to specify order (several calls) should be added IMO.
else that sentence can be removed.

Reply:
I added it to the document because it is sometimes necessary to specify the order when reusing properties defined before.
By the way, the reason for sorting by name is that the order of the lua table is not always the same.